ui/components/component-library/avatar-icon/avatar-icon.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { AvatarBase } from '../avatar-base';
+import Box from '../../ui/box/box';
+import { Icon } from '../icon';
+import {
+  COLORS,
+  BORDER_COLORS,
+  SIZES,
+  DISPLAY,
+  ALIGN_ITEMS,
+  JUSTIFY_CONTENT,
+} from '../../../helpers/constants/design-system';
```

Comment:
nit: order of imports

```suggestion
import {
  COLORS,
  BORDER_COLORS,
  SIZES,
  DISPLAY,
  ALIGN_ITEMS,
  JUSTIFY_CONTENT,
} from '../../../helpers/constants/design-system';

import Box from '../../ui/box/box';

import { Icon } from '../icon';
import { AvatarBase } from '../avatar-base';
```